mkt/extensions/models.py

```diff
@@ -36,10 +36,11 @@
 
 class ExtensionManager(ManagerBase):
     def pending(self):
-        return self.filter(versions__status=STATUS_PENDING).order_by('id')
+        return self.filter(
+            disabled=False, versions__status=STATUS_PENDING).order_by('id')
 
     def public(self):
-        return self.filter(status=STATUS_PUBLIC).order_by('id')
+        return self.filter(disabled=False, status=STATUS_PUBLIC).order_by('id')
 
```

Comment:
I had one originally, but it wasn't used except here so I removed it. Don't want to preliminary refactor things that don't need to be yet.